src/components/shop/scrollable-list/scrollable-list-covers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @module core/layout/scrollable-list-covers
+ * @copyright BBC 2021
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import { getPanelY } from "./scrollable-panel.js";
+
+const getCoverPositions = (scene, covers, padding) => {
+    const safeArea = getPanelY(scene);
+    return {
+        top: covers.top
+            ? {
+                  y: safeArea.y - safeArea.height / 2 + padding.y,
+              }
+            : null,
+        bottom: covers.bottom
+            ? {
+                  y: safeArea.y + safeArea.height / 2 - padding.y,
+              }
+            : null,
```

Comment:
no need for ternary - if no cover, value not read?
then don't need to pass in `covers` either

Reply:
it's so you can have top without bottom, or the opposite.